### PR TITLE
Updates lowPU

### PR DESCRIPTION
**Pull request by jeyserma**
Several updates from lowPU to sync with the main branch. LowPU code/scripts still need to be cleaned.

### CODE REVIEW

**Comment by cippy** [comment]
Hi Jan, for cardTool.writeOutput(statOnly=args.statOnly) keep in mind that we have removed that input argument in PR#73.

See here for an example of how to do the statOnly card in the Wmass setup (note that the mass weights appear before, so that they are not skipped)
https://github.com/WMass/WRemnants/blob/main/scripts/combine/setupCombineWMass.py#L87-L92

The idea is that in this way the implementation is set by the user in their own script rather than internally in cardTool, where it would be hardcoded and usually not fit everyone's needs.

In general, we may actually want to write a new utility script to edit existing cards removing everything we don't need, rather than rerunning the setupXX.py script which also produce the TH2 histograms again and is quite slow

File: scripts/combine/setupHighPU_Wmass_mT.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
```
Comment (kdlong) [inline scripts/combine/setupHighPU_Wmass_mT.py:1]: I guess we probably want to either integrate this with the pt_eta script or at least refactor the code to reuse it. I can take a look at doing this. Also going to look at refactoring things so uncertainties on mT can be added more easily

File: scripts/combine/setupLowPU_Z.py [hunk 2]
```diff
@@ -231,10 +250,11 @@ def scale_recoil_hist_to_variations(scale_hist):
             systAxes = ["tensor_axis_0"],
             labelsByAxis = [""],
         )
-        
-    cardTool.addLnNSystematic("CMS_Top", processes=["TTbar"], size=1.06, group="CMS_bkg_norm")
-    cardTool.addLnNSystematic("CMS_VV", processes=["EWK"], size=1.16, group="CMS_bkg_norm")
-    cardTool.addLnNSystematic("CMS_lumi_lowPU", processes=cardTool.allMCProcesses(), size=1.02, group="CMS_lumi_lowPU")
+    '''
```
Comment (kdlong) [inline scripts/combine/setupLowPU_Z.py:253]: In general I'd prefer to just delete things vs. commenting them out

**Comment by kdlong** [comment]
Ok in general I don't see any problem merging, though a little bit of cleanup could be useful

**Pull request merged**